src/demos/checkboxes/checkboxes.demo.html

```diff
@@ -48,59 +48,148 @@
             </div>
         </div>
     </section>
 </form>
 
-<pre><code clr-code-highlight="language-html">
-&lt;form&gt;
-    &lt;section class=&quot;form-block&quot;&gt;
-        &lt;label&gt;Checkboxes&lt;/label&gt;
-        &lt;div class=&quot;form-group&quot;&gt;
-            &lt;label&gt;Default/Stacked checkbox group&lt;/label&gt;
-            &lt;div class=&quot;checkbox&quot;&gt;
-                &lt;input type=&quot;checkbox&quot; id=&quot;checkrads_1&quot;&gt;
-                &lt;label for=&quot;checkrads_1&quot;&gt;Checkbox 1&lt;/label&gt;
-            &lt;/div&gt;
-            &lt;div class=&quot;checkbox&quot;&gt;
-                &lt;input type=&quot;checkbox&quot; id=&quot;checkrads_2&quot; checked&gt;
-                &lt;label for=&quot;checkrads_2&quot;&gt;Checkbox 2&lt;/label&gt;
-            &lt;/div&gt;
-            &lt;div class=&quot;checkbox disabled&quot;&gt;
-                &lt;input type=&quot;checkbox&quot; id=&quot;checkrads_3&quot; disabled&gt;
-                &lt;label for=&quot;checkrads_3&quot;&gt;A disabled and unchecked checkbox&lt;/label&gt;
-            &lt;/div&gt;
-            &lt;div class=&quot;checkbox disabled&quot;&gt;
-                &lt;input type=&quot;checkbox&quot; id=&quot;checkrads_4&quot; disabled checked&gt;
-                &lt;label for=&quot;checkrads_4&quot;&gt;A disabled and checked checkbox&lt;/label&gt;
-            &lt;/div&gt;
-        &lt;/div&gt;
-        &lt;div class=&quot;form-group&quot;&gt;
-            &lt;label&gt;Inline checkbox group&lt;/label&gt;
-            &lt;div class=&quot;checkbox-inline&quot;&gt;
-                &lt;input type=&quot;checkbox&quot; id=&quot;checkrads_5&quot;&gt;
-                &lt;label for=&quot;checkrads_5&quot;&gt;Checkbox 1&lt;/label&gt;
-            &lt;/div&gt;
-            &lt;div class=&quot;checkbox-inline&quot;&gt;
-                &lt;input type=&quot;checkbox&quot; id=&quot;checkrads_6&quot; checked&gt;
-                &lt;label for=&quot;checkrads_6&quot;&gt;Checkbox 2&lt;/label&gt;
+<form #form="ngForm" novalidate>
+    <section class="form-block">
+        <div class="form-group">
+            <label>With ngForm & ngModel</label>
+            <clr-checkbox
+                    name="termsCheckbox"
+                    id="checkrads_11"
+                    [clrInline]="true"
+                    [(ngModel)]="termsAgreement">
+                I agree to the terms and conditions. {{ form.value | json }}
+            </clr-checkbox>
+        </div>
+    </section>
+</form>
+
+<form>
+    <section class="form-block">
+        <div class="form-group">
+            <label>With a list of objects</label>
+            <clr-checkbox *ngFor="let item of list"
+                          [(clrChecked)]="item.running"
+                          [clrDisabled]="item.disabled">
+                {{ item.disabled ? ' is disabled.' : (item.running ? ' is running.' : 'is off.') }}
+            </clr-checkbox>
+        </div>
+    </section>
```

Comment:
this is a gigantic code-block. do we need all of this in the pre > code section below?